docs/csharp/language-reference/builtin-types/reference-types.md

```diff
@@ -80,7 +80,61 @@ for (int i = 0; i < str.Length; i++)
 // Output: t e s t
 ```
 
-String literals are of type `string` and can be written in two forms, quoted and `@`-quoted. Quoted string literals are enclosed in double quotation marks ("):
+### String literals
+
+String literals are of type `string` and can be written in three forms, raw, quoted, and `@`-quoted. Raw string literals are enclosed in a minimum of three double quotation marks ("""):
```

Comment:
I toyed with it both ways. I'll watch and see what happens. I decided on this format because it highlights the newer feature, where needed.

If I reverse the order, I'll want to separate out the escape sequences until after I introduce the raw string literals. Something like:
- quoted strings
- verbatim strings
- raw strings
- escape sequences, if you aren't using raw strings.